wasm/lib/src/browser_module.rs

```diff
@@ -342,6 +347,45 @@ fn browser_prompt(
     Ok(result)
 }
 
+fn browser_load_module(module: PyStringRef, path: PyStringRef, vm: &VirtualMachine) -> PyResult {
+    let weak_vm = weak_vm(vm);
+
+    let mut opts = web_sys::RequestInit::new();
+    opts.method("GET");
+
+    let request = web_sys::Request::new_with_str_and_init(path.as_str(), &opts)
+        .map_err(|err| convert::js_py_typeerror(vm, err))?;
+
+    let window = window();
+    let request_prom = window.fetch_with_request(&request);
+
+    let future = JsFuture::from(request_prom)
+        .and_then(move |val| {
+            let response = val
+                .dyn_into::<web_sys::Response>()
+                .expect("val to be of type Response");
+            response.text()
+        })
+        .and_then(move |text_value: Promise| {
+            // Convert this other `Promise` into a rust `Future`.
+            JsFuture::from(text_value)
```

Comment:
Instead of a closure, this can just be `JsFuture::from`

Reply:
I am not sure I understand the comment. I am using `JsFuture::from`

Reply:
Instead of `move |text| {...}`, you can just put `JsFuture::from`, because the closure just forwards all it's arguments to `from`, so the closure isn't necessary.

Reply:
Done